### ISSUE TITLE
(maybe) Add ecologits to metrics

### ISSUE THREAD

**Issue by recursix** [ServiceNow/AgentLab#199]
It seems to be low effort to add this metric to our stats
We should look into it. 
https://ecologits.ai/latest/

**Comment by recursix** [comment]
Thanks for engaging please do so :)
ªŁ×
